src/main/java/cloud/fogbow/as/core/systemidp/DefaultSystemRolePlugin.java

```diff
@@ -0,0 +1,59 @@
+package cloud.fogbow.as.core.systemidp;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import cloud.fogbow.as.constants.ConfigurationPropertyKeys;
+import cloud.fogbow.as.constants.SystemConstants;
+import cloud.fogbow.as.core.PropertiesHolder;
+import cloud.fogbow.common.models.SystemUser;
+
+public class DefaultSystemRolePlugin implements SystemRolePlugin {
```

Comment:
Add a new implementation that leaves the role field of the SystemUser null, and call it DefaultSystemRolePlugin. Change the name of this implementation (perhaps VanillaSystemRolePlugin?).

Reply:
Done.